### PR TITLE
docs: add Chinese README

### PR DESCRIPTION
**Pull request by lhdjply**
Help us review this PR! Anyone can [approve it or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### Description of the feature or fix

chore: add Chinese README
https://github.com/lhdjply/lvgl/blob/add_readme_zh/docs/README_zh.rst
@FASTSHIFT  @XuNeo @W-Mai 

> （帮忙看看，有没有翻译错误的地方）

### Checkpoints
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)

### CODE REVIEW

**Comment by W-Mai** [comment]
There's a problem with CI, but it doesn't seem to be your problem
image

cc @PGNetHun

**Comment by lhdjply** [comment]
How to handle this error? Is it necessary to have an rst file?

**Comment by W-Mai** [comment]
After reading other doc files and the submission records of `README_zh.rst.back`, the documents should be presented as `.rst` files.

**Comment by lhdjply** [comment]
> After reading other doc files and the submission records of `README_zh.rst.back`, the documents should be presented as `.rst` files.

@W-Mai I don't know the syntax of rst, can you help me convert it?

File: docs/README_zh.md [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+## :ledger: 概况与总览
+
+**成熟且知名**<br>
+LVGL是最流行的免费开源嵌入式图形库，可以为任何MCU、MPU和显示类型创建漂亮的UI。它得到了行业领先供应商和项目的支持，如Arm、STM32、NXP、Espressif、Nuvoton、Arduino、RT-Thread、Zephyr、NuttX、Adafruit等。
+
+**功能丰富**<br>
+它拥有创建现代美观GUI的所有功能：30多个内置控件、强大的样式系统、Web启发的布局管理器和支持多种语言的排版系统。要将LVGL集成到您的平台中，您只需要至少32kB RAM和128kB Flash、C编译器、帧缓冲区和至少1/10屏幕大小的渲染缓冲区。
+
+**UI编辑器**<br>
+SquareLine Studio是一款专业但价格合理的LVGL拖放UI编辑器。它也在Windows、Linux和MacOS上运行，即使不在网站上注册，你也可以尝试一下。
```
Comment (W-Mai) [inline docs/README_zh.md]: 无需注册，免费试用

**Comment by W-Mai** [comment]
> > After reading other doc files and the submission records of `README_zh.rst.back`, the documents should be presented as `.rst` files.
> 
> @W-Mai I don't know the syntax of rst, can you help me convert it?

我也没有写过rst文件，我觉得你可以在原来的rst文件的基础上进行修改
I haven't written a `.rst` file either. I think you can modify it on the basis of the original `.rst` file. And mv it from .rst.back to .rst

**Comment by W-Mai** [comment]
@kisvegabor 
Did you have any problems with the Chinese document provider before? 
What is the reason why Chinese README has not been provided? 
Do we need to use `.rst` to write documents? 😁

**Comment by lhdjply** [comment]
The V8 version is a Chinese document. V9 seems to still have the homepage button, but when clicked on the page, it shows 404.

**Comment by W-Mai** [comment]
> The V8 version is a Chinese document. V9 seems to still have the homepage button, but when clicked on the page, it shows 404.

sure.

**Comment by lhdjply** [comment]
I have convert md to rst file

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <a href="https://github.com/sponsors/lvgl" target="_blank"><img align="left" src="https://lvgl.io/assets/images/sponsor.png" height="32px"></a>
 
 <p align="right">
-  <b>English</b> | <a href="./docs/README_zh.md">中文</a> | <a href="./docs/README_pt_BR.md">Português do Brasil</a> | <a href="./docs/README_jp.md">日本語</a>
+  <b>English</b> | <a href="./docs/README_zh.rst">中文</a> | <a href="./docs/README_pt_BR.md">Português do Brasil</a> | <a href="./docs/README_jp.md">日本語</a>
```
Comment (W-Mai) [inline README.md:5]:
```suggestion
  <b>English</b> | <a href="./docs/README_zh.md">中文</a> | <a href="./docs/README_pt_BR.md">Português do Brasil</a> | <a href="./docs/README_jp.md">日本語</a>
```

md is ok, md is built from *.rst

File: docs/README_zh.rst [hunk 3]
```diff
@@ -0,0 +1,616 @@
...
+.. _rocket-特性:
+
+🚀 特性
+-------
+
+**免费和可移植性**
+
+-  一个完全可移植的C（C++兼容）库，没有外部依赖关系。
+-  可以编译到任何MCU或MPU，使用任何RTOS或者操作系统。
+-  支持单色、ePaper、OLED或TFT显示器，甚至显示器。
```
Comment (MouriNaruto) [inline docs/README_zh.rst]: 这里读起来也有点奇妙，根据 https://english.stackexchange.com/questions/475894/difference-between-screen-display-monitor-and-display-monitor 里面关于 display 和 monitor 的区别，也许也能改进改进

File: docs/README_zh.rst [hunk 4]
```diff
@@ -0,0 +1,616 @@
...
+
+.. _rocket-特性:
+
+🚀 特性
+-------
+
+**免费和可移植性**
+
+-  一个完全可移植的C（C++兼容）库，没有外部依赖关系。
+-  可以编译到任何MCU或MPU，使用任何RTOS或者操作系统。
```
Comment (MouriNaruto) [inline docs/README_zh.rst]: 按照 7-Zip 中文官网翻译者曾经写的[中英文混合排版指导](https://github.com/sparanoid/chinese-copywriting-guidelines)，我感觉你看了以后应该还能继续改进

**Comment by kisvegabor** [comment]
The CI should pass now, please rebase on `master`.

**Review by kisvegabor — DISMISSED** [review]

**Comment by lhdjply** [comment]
> The CI should pass now, please rebase on `master`.

Done

File: docs/README_zh.rst [hunk 5]
```diff
@@ -0,0 +1,616 @@
...
+-  支持单色、ePaper、OLED或TFT显示器，甚至显示器。
+   `移植指南 <https://docs.lvgl.io/master/porting/project.html>`__
+-  根据MIT许可证，因此您也可以在商业项目中轻松使用它。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  该项目使用MIT许可证，因此您可以在商业项目中轻松使用它。
```

File: docs/README_zh.rst [hunk 6]
```diff
@@ -0,0 +1,616 @@
...
+-  只需要32kB的RAM和128kB的Flash，一个帧缓冲区，以及至少1/10屏幕大小的渲染缓冲区。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  仅需32kB RAM和128kB Flash，一个帧缓冲区，以及至少1/10屏幕大小的渲染缓冲区。
```

File: docs/README_zh.rst [hunk 7]
```diff
@@ -0,0 +1,616 @@
...
+-  支持操作系统、外部存储器和GPU，但不是必需的。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  支持使用可选的操作系统、外部存储器和GPU。
```

File: docs/README_zh.rst [hunk 8]
```diff
@@ -0,0 +1,616 @@
...
+
+**控件、样式、布局等**
+
+-  30+内置\ `控件 <https://docs.lvgl.io/master/widgets/index.html>`__:
+    按钮、标签、滑块、图表、键盘、仪表、弧形、表格等等。
+-  灵活的\ `样式系统 <https://docs.lvgl.io/master/overview/style.html>`__
+   具有约100个样式属性，可在任何状态下自定义小部件的任何部分。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
   支持约100个样式属性，可在任何状态下自定义小控件的任何部分。
```

File: docs/README_zh.rst [hunk 9]
```diff
@@ -0,0 +1,616 @@
...
+-  `Flexbox <https://docs.lvgl.io/master/layouts/flex.html>`__ and
+   `网格 <https://docs.lvgl.io/master/layouts/grid.html>`__
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
   `Grid布局 <https://docs.lvgl.io/master/layouts/grid.html>`__
```

File: docs/README_zh.rst [hunk 10]
```diff
@@ -0,0 +1,616 @@
...
+-  只需要32kB的RAM和128kB的Flash，一个帧缓冲区，以及至少1/10屏幕大小的渲染缓冲区。
+-  支持操作系统、外部存储器和GPU，但不是必需的。
+
+**控件、样式、布局等**
+
+-  30+内置\ `控件 <https://docs.lvgl.io/master/widgets/index.html>`__:
+    按钮、标签、滑块、图表、键盘、仪表、弧形、表格等等。
+-  灵活的\ `样式系统 <https://docs.lvgl.io/master/overview/style.html>`__
+   具有约100个样式属性，可在任何状态下自定义小部件的任何部分。
+-  `Flexbox <https://docs.lvgl.io/master/layouts/flex.html>`__ and
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  `Flex布局 <https://docs.lvgl.io/master/layouts/flex.html>`__ 和
```

File: docs/README_zh.rst [hunk 11]
```diff
@@ -0,0 +1,616 @@
...
+   `网格 <https://docs.lvgl.io/master/layouts/grid.html>`__
+   像布局引擎一样，以响应的方式自动调整小部件的大小和位置。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
   可以响应式自动调整小控件的大小和位置。
```

File: docs/README_zh.rst [hunk 12]
```diff
@@ -0,0 +1,616 @@
...
+-  文本以UTF-8编码呈现，支持CJK、泰语、印地语、阿拉伯语和波斯语书写系统。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  文本支持UTF-8编码，支持CJK、泰语、印地语、阿拉伯语和波斯语书写系统。
```

File: docs/README_zh.rst [hunk 13]
```diff
@@ -0,0 +1,616 @@
...
+-  换行、紧排、文本滚动、亚像素渲染、拼音输入法、文本表情符号。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  支持自动换行、字距调整、文本滚动、亚像素渲染、拼音输入法、文本表情符号。
```

**Comment by W-Mai** [comment]
Wait a moment to merge, I'm polishing some small translations.😁

File: docs/README_zh.rst [hunk 14]
```diff
@@ -0,0 +1,616 @@
...
+-  渲染引擎支持动画、抗锯齿、不透明度、平滑滚动、阴影、图像转换等。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  渲染引擎支持动画、抗锯齿、不透明度、平滑滚动、阴影、图形变换等。
```

File: docs/README_zh.rst [hunk 15]
```diff
@@ -0,0 +1,616 @@
...
+-  支持鼠标、触摸板、小键盘、键盘、外部按钮、编码器\ `输入设备 <https://docs.lvgl.io/master/porting/indev.html>`__\ 。
+-  支持\ `多显示器 <https://docs.lvgl.io/master/overview/disp.html#multiple-display-support>`__\ 。
+
+**绑定和构建支持**
+
+-  `Microython绑定 <https://blog.lvgl.io/2019-02-20/micropython-bindings>`__
+   公开LVGL API
+-  `PikaScript绑定 <https://blog.lvgl.io/2022-08-24/pikascript-and-lvgl>`__
+   python在MCU上更轻更容易。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
   在MCU上的更轻更简单的Python版本
```

File: docs/README_zh.rst [hunk 16]
```diff
@@ -0,0 +1,616 @@
...
+-  未使用自定义生成系统。您可以在构建项目的其他文件时构建LVGL。
+-  对Make和\ `CMake <https://docs.lvgl.io/master/get-started/platforms/cmake.html>`__\ 的支持是开箱即用的。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  支持开箱即用的Make和\ `CMake <https://docs.lvgl.io/master/get-started/platforms/cmake.html>`__\ 编译系统。
```

File: docs/README_zh.rst [hunk 17]
```diff
@@ -0,0 +1,616 @@
...
+-  在\ `PC上开发 <https://docs.lvgl.io/master/get-started/platforms/pc-simulator.html>`__\ ，并在嵌入式硬件上使用相同的UI代码。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  支持在\ `PC上开发 <https://docs.lvgl.io/master/get-started/platforms/pc-simulator.html>`__\ ，并可以在嵌入式硬件上使用相同的UI代码。
```

File: docs/README_zh.rst [hunk 18]
```diff
@@ -0,0 +1,616 @@
...
+-  使用我们的\ `Emscripten端口 <https://github.com/lvgl/lv_web_emscripten>`__\ 将C写的UI代码转换为HTML文件。
```
Comment (W-Mai) [inline docs/README_zh.rst]:
```suggestion
-  支持使用我们的\ `Emscripten移植 <https://github.com/lvgl/lv_web_emscripten>`__\ 从而将C写的UI代码转换为HTML文件。
```

**Review by W-Mai — CHANGES_REQUESTED** [review]

**Comment by kisvegabor** [comment]
Okay :slightly_smiling_face: 

BTW, why did you feel better to use `rst` as README?

**Comment by W-Mai** [comment]
> Okay 🙂
> 
> BTW, why did you feel better to use `rst` as README?

I saw that the change 7 months ago was to change readme to rst, so...

I just followed the changes of my predecessors, and I have no opinion on this.

**Comment by lhdjply** [comment]
@W-Mai  Thanks

**Comment by W-Mai** [comment]
I'm done for this.♥️

File: docs/README_zh.rst [hunk 19]
```diff
@@ -0,0 +1,615 @@
...
+-------
+
+**免费和可移植性**
+
+-  一个完全可移植的C（C++兼容）库，没有外部依赖关系。
+-  可以编译到任何MCU或MPU，使用任何RTOS或者操作系统。
+-  支持单色、ePaper、OLED或TFT显示器，甚至显示器。
+   `移植指南 <https://docs.lvgl.io/master/porting/project.html>`__
+-  该项目使用MIT许可证，因此您可以在商业项目中轻松使用它。
+-  仅需32kB RAM和128kB Flash，一个帧缓冲区，以及至少1/10屏幕大小的渲染缓冲区。
```
Comment (MouriNaruto) [inline docs/README_zh.rst]: 这边的 kB 是否需要改成 KB 啥的

File: docs/README_zh.rst (hunk 4 above)
Comment (MouriNaruto) [reply docs/README_zh.rst]: 主要是中英文之间加个空格观感会好很多

File: docs/README_zh.rst (hunk 19 above)
Comment (lhdjply) [reply docs/README_zh.rst]: 好

File: docs/README_zh.rst [hunk 20]
```diff
@@ -0,0 +1,614 @@
...
+   如用户界面设计、实施和咨询，使 UI 开发更简单、更快。
+
+.. _heart-赞助:
+
+❤️ 赞助
+-------
+
+如果 LVGL 为您节省了大量时间和金钱，或者您只是在使用它时玩得很开心，请考虑\ `支持它的开发 <https://github.com/sponsors/lvgl>`__\ 。
+
+**我们为什么要捐款？**\
```
Comment (MouriNaruto) [inline docs/README_zh.rst]: 按照英文原文 How do we spend the donations? 应该翻译成 我们如何使用捐款？
目前的意思看起来挺奇怪的，变成 LVGL 那群人对外捐款了（摊手）

File: docs/README_zh.rst [hunk 21]
```diff
@@ -0,0 +1,614 @@
...
+
+**我们为什么要捐款？**\ 
+
+我们的目标是为 LVGL 做得最多的人提供经济补偿。这意味着不仅维护人员，而且任何实现伟大功能的人都应该从累积的资金中获得报酬。我们用捐款来支付服务器和相关服务等运营成本。
+
+**如何捐赠？**\  
+
+我们使用\ `GitHub Sponsors <https://github.com/sponsors/lvgl>`__\ ，您可以轻松发送一次性或定期捐款。您还可以以透明的方式查看我们的所有费用。
+
+**如何获得捐款报酬？**\
```
Comment (MouriNaruto) [inline docs/README_zh.rst]: 其实感觉应该改成 如何从您的贡献中获得报酬？这样的描述……目前的意思变成了向 LVGL 捐赠还能拿提成的感觉（摊手）

File: docs/README_zh.rst (hunk 20 above)
Comment (MouriNaruto) [reply docs/README_zh.rst]: 其实应该是我们如何使用捐赠（donations 还是改成捐赠会柔和一些）

File: docs/README_zh.rst [hunk 22]
```diff
@@ -0,0 +1,614 @@
...
+   slider.set_style_outline_width(3, lv.PART.KNOB)
+   slider.set_style_outline_pad(-5, lv.PART.KNOB)
+   slider.set_style_shadow_spread(2, lv.PART.KNOB)
+
+.. raw:: html
+
+   </details>
+   <br>
+
+英语、希伯来语（混合 LTR-RTL ）和中文
```
Comment (MouriNaruto) [inline docs/README_zh.rst]: 这边的（混合 LTR-RTL ）其实写成 双向文本排版 会更好一些
https://zh.wikipedia.org/wiki/%E9%9B%99%E5%90%91%E6%96%87%E7%A8%BF

**Review by MouriNaruto — APPROVED** [review]

**Review by FASTSHIFT — APPROVED** [review]

**Comment by FASTSHIFT** [comment]
Merged, thank you all very much~

**Pull request merged**